### PR TITLE
Fix `--language` option.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Fixes https://github.com/caskroom/homebrew-cask/issues/35448.

### CODE REVIEW

File: Library/Homebrew/brew.rb [hunk 1]
```diff
@@ -13,21 +13,14 @@
 $:.unshift(HOMEBREW_LIBRARY_PATH.to_s)
 require "global"
 require "tap"
+require "utils"
 
 if ARGV == %w[--version] || ARGV == %w[-v]
   puts "Homebrew #{HOMEBREW_VERSION}"
   puts "Homebrew/homebrew-core #{CoreTap.instance.version_string}"
   exit 0
 end
```
Comment (MikeMcQuaid) [inline Library/Homebrew/brew.rb]: Can you benchmark the performance before/after this change on `--version`? Thanks!

Comment (reitermarkus) [inline Library/Homebrew/brew.rb]: There's no measurable difference.

Comment (reitermarkus) [inline Library/Homebrew/brew.rb]: Well, we are already requiring it in `global` anyway.

**Comment by bunam** [comment]
no really fixed

`brew cask install --force libreoffice libreoffice-language-pack --language=fr
Error: invalid option: --language=fr`

**Pull request merged**